src/sentry/auth/superuser.py

```diff
@@ -85,7 +87,24 @@ def get_superuser_scopes(auth_state: RpcAuthState, user: Any):
     return superuser_scopes
 
 
-def is_active_superuser(request):
+def superuser_has_permission(request: HttpRequest | Request):
```

Comment:
nit: return type bool here!

```suggestion
def superuser_has_permission(request: HttpRequest | Request) -> bool:
```